pkgs/tools/misc/didyoumean/default.nix

```diff
@@ -2,32 +2,54 @@
 , stdenv
 , rustPlatform
 , fetchFromGitHub
+, installShellFiles
+, pkg-config
 , libxcb
-# Darwin dependencies
+, openssl
+  # Darwin dependencies
 , AppKit
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "didyoumean";
-  version = "1.1.0";
+  version = "1.1.3";
 
   src = fetchFromGitHub {
     owner = "hisbaan";
     repo = "didyoumean";
     rev = "v${version}";
-    sha256 = "sha256-t2bmvz05vWIxQhC474q/9uky1kAQoFN8Z+qflw5Vj68=";
+    sha256 = "sha256-hHl9PGNDFN7Dad2JOlAy99dz0pC9OmphwYMJHBBwx7Y=";
   };
 
-  cargoSha256 = "sha256-4DbziI9enib4pm9/P4WEu15glIxtejaV2GCqbzuxxyw=";
+  cargoSha256 = "sha256-rjkj9MO6fXVOk3fA87olGt/iIaJ8Zv/cy/Cqy/pg6yI=";
 
-  buildInputs = lib.optional stdenv.isLinux [ libxcb ]
-    ++ lib.optionals stdenv.isDarwin [ AppKit ];
+  nativeBuildInputs = [
+    installShellFiles
+  ] ++ lib.optionals stdenv.isLinux [
+    pkg-config
+  ];
+
+  buildInputs = lib.optionals stdenv.isLinux [
+    libxcb
+    openssl
+  ] ++ lib.optionals stdenv.isDarwin [
+    AppKit
+  ];
+
+  postInstall = ''
+    installManPage man/dym.1
+    installShellCompletion completions/dym.{bash,fish}
+    installShellCompletion --zsh completions/_dym
+  '';
+
+  # Clipboard doesn't exist in test environment
+  doCheck = false;
```

Comment:
Are _all_ tests broken? Always nice to have some working tests.

Reply:
There is only _one_ test in the project.

Reply:
Ah, too bad then